### PR TITLE
README: suggest installation procedure

### PR DESCRIPTION
**Pull request by hakamine**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,12 +5,39 @@ Fixity is a simple commandline tool that assists in checking fixity for AIPs sto
...
 The Archivematica Storage Service provides a REST API endpoint for checking fixity of AIPs. This tool provides a simple commandline interface to that functionality, in a way that's simple to automate.
 
+
+
+Installation
+------------
+
+- Checkout or link the code to /usr/lib/archivematica/fixity
+- Create a virtualenv in /usr/share/python/fixity, and install fixity and dependencies in it
+  ```
```
Comment (Hwesta) [inline README.md:16]: This needs a blank line before the ``` to display the code as a block.

**Comment by hakamine** [comment]
@Hwesta thank you! (I was wondering why it was not showing correctly on github despite displaying properly in the MD viewer of my text editor)

File: README.md [hunk 2]
```diff
@@ -5,12 +5,45 @@ Fixity is a simple commandline tool that assists in checking fixity for AIPs sto
...
+
+Installation
+------------
+
+- Checkout or link the code to /usr/lib/archivematica/fixity
+- Create a virtualenv in /usr/share/python/fixity, and install fixity and dependencies in it
+
+  ```
+  # virtualenv /usr/share/python/fixity
+  # source /usr/share/python/fixity/bin/activate
```
Comment (Hwesta) [inline README.md]: I think this need something to make clear this is running as root and not just a comment. Maybe prepend root@host? Or change `#` to `$`? Or remove the command prompts & virtualenv stuff?

**Comment by hakamine** [comment]
Good point. I think it may be easier to understand with the command prompts included than without. Added the user@host to the prompts as suggested

**Comment by Hwesta** [comment]
:+1:

**Pull request merged**